config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
   root "food_trucks#index"
-  devise_for :users
+  devise_for :users, controllers: {
+        sessions: 'users/sessions'
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.